include/trampolines.h

```diff
@@ -85,9 +93,17 @@ dmtcp_setup_trampoline_by_addr(void *addr,
 
   /* Base address of page where func resides. */
   page_base = (void *)((unsigned long)info->addr & pagemask);
-
+  int pagecount = 1;
+
+  /* Increase the pagecount if number of bytes needs to be wriiten
+   * to set a trampoline falls in the next page.
```

Comment:
Suggested re-wording:  ... if _the_ number of bytes _that_ needs to be written _for_ the trampoline _happens to fall_ in the next page.

Reply:
Thanks for pointing out :)